lib/src/theme.dart

```diff
@@ -410,6 +410,57 @@ class NesOverlayTransitionTheme
   }
 }
 
+/// {@template nes_progress_bar_theme}
+/// Class with information regarding [NesProgressBar] inside NesUI.
+/// {@endtemplate}
+class NesProgressBarTheme extends ThemeExtension<NesProgressBarTheme> {
+  /// {@macro nes_progress_bar_theme}
+  const NesProgressBarTheme({
+    required this.background,
+    required this.color,
+  });
+
+  /// The color for the progress bar background.
+  /// Defaults to [ColorScheme.surface]
+  final Color background;
```

Comment:
By looking on the code, I think this is incorrect? We are using the `textTheme.bodyMedium?.color` right?

Reply:
fixed the defaults.